frontend/src/App.jsx

```diff
@@ -28,6 +28,11 @@ function App() {
             <ProtectedRoute>
               <Posts />
             </ProtectedRoute>
+          } />
+           <Route path="/company" element={
```

Comment:
this has to be `/businesses` to line up with the routes we already have. Otherwise it will not work.

Reply:
nvm i see that you changed it inside of the navbar as well then